src/libtools/elements.cc

```diff
@@ -148,7 +148,7 @@ bool Elements::isEleFull(std::string fullname) {
 }
 
 bool Elements::isEleShort(std::string shortname) {
-  if (!this - _filled_EleShort) {
+  if (!this->_filled_EleShort) {
```

Comment:
I still do not get it, you should fill FillEleShort not long, this is confusing. I do not understand

Reply:
I get the part with the function name, but the bools should not disagree

Reply:
Oh, right this one is backwards.